src/app/components/file-upload/file-upload.component.ts

```diff
@@ -1,40 +1,156 @@
 /* eslint-disable no-console */
 /* eslint-disable no-undef */
-import { Component, OnInit } from '@angular/core';
+import {
+  Component, OnInit, ViewChild, ElementRef
+} from '@angular/core';
 import { FormBuilder, FormGroup } from '@angular/forms';
 import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import yaml from 'js-yaml';
+
+// NgRX
+import { Store } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { SwagPaths } from '../../state/swag-paths/swag-paths.model';
+import { POPULATE_PATHS } from '../../state/swag-paths/swag-paths.actions';
 
 @Component({
   selector: 'app-file-upload',
   templateUrl: './file-upload.component.html',
   styleUrls: ['./file-upload.component.scss'],
 })
 export class FileUploadComponent implements OnInit {
-  private baseUrl = 'http://localhost:8083/Docutest';
+  // NgRX
+  swagPaths$: Observable<SwagPaths>;
+
+  private Base_Url = 'http://localhost:8083/Docutest';
```

Comment:
I believe this naming convention should be `baseUrl`. This might be an issue detected by `ng lint`.

Reply:
This issue has yet to be resolved.